src/Forms/HTMLEditor/TinyMCEGZIPGenerator.php

```diff
@@ -11,7 +11,7 @@
 /**
  * Uses the default tiny_mc_gzip.php handler
  *
- * @deprecated 4.0.0:5.0.0
+ * @deprecated 4.0.0
```

Comment:
Add the message here and update the version in the notice below.
Also update the message to something more standardised (in this case it will be removed without equivalent functionality)

Reply:
Updated